Plotting/python/Datacards/AnalysisSpecification.py

```diff
@@ -9,6 +9,7 @@
 from TTH.Plotting.Datacards.AnalysisSpecificationFromConfig import analysisFromConfig
 
 analyses = {}
+#for config in ["config_fh.cfg"]: 
```

Comment:
we probably need here a mechanism to trigger one or the other, to keep you from commenting this line on/off.